### PR TITLE
Docs: Add changelog entry for PR 295

### PR DESCRIPTION
**Pull request by jaredbeck**
[ci skip]

### CODE REVIEW

**Comment by jaredbeck** [comment]
Also, while we're talking about it, could you please do a patch release (currently, that would be 4.3.1) with this fix? Thanks.

File: CHANGELOG [hunk 1]
```diff
@@ -12,7 +12,8 @@ Added
 
 Fixed
 
-- None
+- SQL error in Rails Conditional GET (304 caching)
+  [#295](https://github.com/collectiveidea/audited/pull/295)
```
Comment (domcleal) [inline CHANGELOG]: This file's not Markdown, perhaps just use the number `#295`.

Comment (jaredbeck) [inline CHANGELOG]: What do you think about converting the file to markdown? It displays nicely in github, with clickable links. We could just add a `.md` extension.

Comment (domcleal) [inline CHANGELOG]: Sure, that'd be good.

**Comment by jaredbeck** [comment]
>> What do you think about converting the file to markdown?

> Sure, that'd be good.

File renamed, ready for second review, thanks.

**Comment by domcleal** [comment]
Thanks @jaredbeck, much easier to read. I'll prep a release tomorrow to include #295.

**Pull request merged**